components/tools/OmeroPy/src/omero/__init__.py

```diff
@@ -104,6 +104,14 @@ def proxy_to_instance(proxy_string, default=None):
     """
     import omero
     parts = proxy_string.split(":")
+    # If a default is provided and the format is Class:ID
```

Comment:
The problem is when the default is 'Image' and the argument is 'Fileset:1' then:
```
Fileset:1 ->  Image:1
```
ie the default is used but it seems to me that this should be an error.